pkg/gardenlet/controller/shoot/shoot_control_reconcile.go

```diff
@@ -315,6 +315,16 @@ func (r *shootReconciler) runReconcileShootFlow(ctx context.Context, o *operatio
 			Fn:           flow.TaskFn(botanist.Shoot.Components.ControlPlane.ResourceManager.Wait).SkipIf(o.Shoot.HibernationEnabled),
 			Dependencies: flow.NewTaskIDs(deployGardenerResourceManager),
 		})
+		deployShootNamespaces = g.Add(flow.Task{
+			Name:         "Deploying shoot namespaces system component",
```

Comment:
There is 1 more task with the same name and it is not removed.

Reply:
Sorry, what do you mean?

Reply:
Ah, thank you! Again, nice catch! Thanks for your valuable feedback 🤝